### ISSUE TITLE
[Object Detection API] Show wrong detection

### ISSUE THREAD

**Issue by cannguyen275** [tensorflow/models#5766]
### System information
- **What is the top-level directory of the model you are using**:
models/research/object_detection/
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Linux Ubuntu 17.04
- **TensorFlow installed from (source or binary)**:
conda install
- **TensorFlow version (use command below)**:
1.7.0
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
CUDA® Toolkit 9.0; cuDNN v7.1
- **GPU model and memory**:
GForce GTX 1080Ti (11GB)
- **Exact command to reproduce**:

```
python eval.py --logtostderr --pipeline_config_path=training/faster_rcnn_inception_resnet_v2_atrous_coco.config --checkpoint_dir=training_point/ --eval_dir=eval/
```
```
tensorboard --logdir=training:training_point/,testing:eval/
```
### Describe the problem

I've trained a model using the object detection API. Using eval.py, I can evaluate my model with my validation set. The mAP is ~96%.
I want to see the wrong prediction. It is 4% left and my validation set is 10000.
Is there any way to see that?
Thanks,

**Comment by pkulzc** [comment]
There isn't a obvious way to only select the false positives. But you can use our jupyter notebook to run the model you trained through all images and get visualizations of both detections and groundtruths.

**Comment by jillelajitta** [comment]
@pkulzc 
I parsed out 90 classes from OPEN IMAGE DATASET for object detection. I roughly have 600k train images and 200k val images.
I have trained this dataset on coco pretrained inception v2 model for 400k steps,at 200k steps it is only detecting smaller objects, after 300k steps it is only detecting larger objects. Could you please help me where I am going wrong.
How many steps do you suggest me to train? What is the best learning rate to start with.

**Comment by cannguyen275** [comment]
@pkulzc That should be a good feature to have. Hope we can have that in the future.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
